<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>652</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(71, 71, 72);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="top_bar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(49, 48, 49);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_toggle">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>70</width>
         <height>36</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(39, 38, 39);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="menu_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	color: rgb(255,255,255);
}</string>
          </property>
          <property name="text">
           <string>Menu</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="Content">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>-1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_left_menu">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(39, 38 ,39);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignTop">
           <widget class="QFrame" name="frame_top_menus">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="Btn_1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	color: rgb(255,255,255);
	background_color: rgb(39,38,39);
}
QPushButton:hover{
	background-color: rgb(77, 77, 78);
}</string>
               </property>
               <property name="text">
                <string>View All</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Btn_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	color: rgb(255,255,255);
	background_color: rgb(39,38,39);
}
QPushButton:hover{
	background-color: rgb(77, 77, 78);
}</string>
               </property>
               <property name="text">
                <string>Check</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Btn_3">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	color: rgb(255,255,255);
	background_color: rgb(39,38,39);
}
QPushButton:hover{
	background-color: rgb(77, 77, 78);
}</string>
               </property>
               <property name="text">
                <string>Fix</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Btn_4">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	color: rgb(255,255,255);
	background_color: rgb(39,38,39);
}
QPushButton:hover{
	background-color: rgb(77, 77, 78);
}</string>
               </property>
               <property name="text">
                <string>Add New</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_pages">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="Page_widgets">
            <widget class="QWidget" name="page_1">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <widget class="QLabel" name="page_1_label">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>120</y>
                <width>611</width>
                <height>371</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	color:rgb(255,255,255);
	font: 30pt &quot;.AppleSystemUIFont&quot;;
}

</string>
              </property>
              <property name="text">
               <string>Main Page</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="page_2">
             <widget class="QTableWidget" name="table_CVE">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>140</y>
                <width>671</width>
                <height>351</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QTable{
	color: rgb(255,255,255);
	font: 60pt &quot;.AppleSystemUIFont&quot;;
}</string>
              </property>
              <row>
               <property name="text">
                <string>1</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>CVE</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Description</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>CVSS</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>link</string>
               </property>
              </column>
              <item row="0" column="0">
               <property name="text">
                <string>CVE-2019-14271 </string>
               </property>
              </item>
             </widget>
             <widget class="QLabel" name="page_2_label">
              <property name="geometry">
               <rect>
                <x>-1</x>
                <y>100</y>
                <width>531</width>
                <height>31</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	color:rgb(255,255,255);
	font: 15pt &quot;.AppleSystemUIFont&quot;;
}</string>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="page_3">
             <widget class="QLabel" name="page_3_label">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>130</y>
                <width>311</width>
                <height>41</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	color:rgb(255,255,255);
	font: 15pt &quot;.AppleSystemUIFont&quot;;
}</string>
              </property>
              <property name="text">
               <string>Page 3</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_vul1">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>180</y>
                <width>180</width>
                <height>35</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	color:rgb(255,255,255);
}</string>
              </property>
              <property name="text">
               <string>Vul 1</string>
              </property>
             </widget>
             <widget class="QPushButton" name="Btn_vul1">
              <property name="geometry">
               <rect>
                <x>250</x>
                <y>190</y>
                <width>113</width>
                <height>32</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	color:rgb(255,255,255);
}

QPushButton:hover{
	background-color: rgb(77, 77, 78);
}</string>
              </property>
              <property name="text">
               <string>Check</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_vul2">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>220</y>
                <width>180</width>
                <height>35</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	color:rgb(255,255,255);
}</string>
              </property>
              <property name="text">
               <string>Vul 2</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_vul3">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>260</y>
                <width>180</width>
                <height>35</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	color:rgb(255,255,255);
}</string>
              </property>
              <property name="text">
               <string>Vul 3</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_vul4">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>300</y>
                <width>180</width>
                <height>35</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	color:rgb(255,255,255);
}</string>
              </property>
              <property name="text">
               <string>Vul 4</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_vul5">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>340</y>
                <width>180</width>
                <height>35</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	color:rgb(255,255,255);
}</string>
              </property>
              <property name="text">
               <string>Vul 5</string>
              </property>
             </widget>
             <widget class="QPushButton" name="Btn_vul2">
              <property name="geometry">
               <rect>
                <x>250</x>
                <y>230</y>
                <width>113</width>
                <height>32</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	color:rgb(255,255,255);
}

QPushButton:hover{
	background-color: rgb(77, 77, 78);
}</string>
              </property>
              <property name="text">
               <string>Check</string>
              </property>
             </widget>
             <widget class="QPushButton" name="Btn_vul3">
              <property name="geometry">
               <rect>
                <x>250</x>
                <y>270</y>
                <width>113</width>
                <height>32</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	color:rgb(255,255,255);
}

QPushButton:hover{
	background-color: rgb(77, 77, 78);
}</string>
              </property>
              <property name="text">
               <string>Check</string>
              </property>
             </widget>
             <widget class="QPushButton" name="Btn_vul5">
              <property name="geometry">
               <rect>
                <x>250</x>
                <y>350</y>
                <width>113</width>
                <height>32</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	color:rgb(255,255,255);
}

QPushButton:hover{
	background-color: rgb(77, 77, 78);
}</string>
              </property>
              <property name="text">
               <string>Check</string>
              </property>
             </widget>
             <widget class="QPushButton" name="Btn_vul4">
              <property name="geometry">
               <rect>
                <x>250</x>
                <y>310</y>
                <width>113</width>
                <height>32</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	color:rgb(255,255,255);
}

QPushButton:hover{
	background-color: rgb(77, 77, 78);
}</string>
              </property>
              <property name="text">
               <string>Check</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="page_4">
             <widget class="QLabel" name="page_4_label">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>110</y>
                <width>111</width>
                <height>41</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	color:rgb(255,255,255);
	font: 15pt &quot;.AppleSystemUIFont&quot;;
}</string>
              </property>
              <property name="text">
               <string>Available Fixes</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
             <widget class="QPushButton" name="Btn_fix">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>470</y>
                <width>113</width>
                <height>32</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	color: rgb(255,255,255);
}
QPushButton:hover{
	background-color: rgb(77, 77, 78);
}</string>
              </property>
              <property name="text">
               <string>Fix</string>
              </property>
             </widget>
             <widget class="QRadioButton" name="radioButton_1">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>140</y>
                <width>190</width>
                <height>60</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QRadioButton{
	color: rgb(255,255,255);
}</string>
              </property>
              <property name="text">
               <string>CVE-2019-14271 </string>
              </property>
             </widget>
             <widget class="QRadioButton" name="radioButton_2">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>200</y>
                <width>190</width>
                <height>60</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QRadioButton{
	color: rgb(255,255,255);
}</string>
              </property>
              <property name="text">
               <string>CVE-2019-13139</string>
              </property>
             </widget>
             <widget class="QRadioButton" name="radioButton_3">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>260</y>
                <width>190</width>
                <height>60</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QRadioButton{
	color: rgb(255,255,255);
}</string>
              </property>
              <property name="text">
               <string>CVE-2019-5736</string>
              </property>
             </widget>
             <widget class="QRadioButton" name="radioButton_4">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>320</y>
                <width>190</width>
                <height>60</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QRadioButton{
	color: rgb(255,255,255);
}</string>
              </property>
              <property name="text">
               <string>CVE-4-</string>
              </property>
             </widget>
             <widget class="QRadioButton" name="radioButton_5">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>380</y>
                <width>190</width>
                <height>60</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QRadioButton{
	color: rgb(255,255,255);
}</string>
              </property>
              <property name="text">
               <string>CVE-5-</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="page_5">
             <widget class="QLabel" name="page_5_label">
              <property name="geometry">
               <rect>
                <x>110</x>
                <y>140</y>
                <width>381</width>
                <height>211</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	color:rgb(255,255,255);
	font: 30pt &quot;.AppleSystemUIFont&quot;;
}
</string>
              </property>
              <property name="text">
               <string>Page 5</string>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
